soh/src/code/audio_load.c

```diff
@@ -1366,16 +1368,23 @@ void AudioLoad_Init(void* heap, size_t heapSize) {
     int startingSeqNum = MAX_AUTHENTIC_SEQID; // 109 is the highest vanilla sequence
```

Comment:
is `MAX_AUTHENTIC_SEQID` a lie? are there authentic sequences with higher ids than 109?

Reply:
No, the sequences end there, but `AudioCollection` has more than just sequences in it, like the sound fonts at 130-136 which were the original issue.

Reply:
It's a bit cumbersome to have a sequence map in `audio_load.c` and `AudioCollection`, but I'm not sure how to unify them.

Reply:
So yeah, as mentioned elsewhere, it's more than just the instruments in there. the sound effects are added in as well later on.